test/colony-funding.js

```diff
@@ -403,4 +406,290 @@ contract("Colony Funding", () => {
       assert.equal(nonRewardPotsTotal.toNumber(), 297);
     });
   });
+
+  describe("when creating reward payouts", async () => {
+    const initialFunding = toBN("9").mul(toBN(10).pow(toBN(75)));
+    const totalReputation = 80 * 1e18;
+    const userReputation1 = 50 * 1e18;
+    const userReputation2 = 30 * 1e18;
+    const userAddress1 = addresses[0];
+    const userAddress2 = addresses[1];
+    const userAddress3 = addresses[2];
+
+    beforeEach(async () => {
+      await fundColonyWithTokens(colony, otherToken, initialFunding.toString());
+      await colony.mintTokens(initialFunding.toString());
+      await colony.bootstrapColony([userAddress1, userAddress2], [userReputation1, userReputation2]);
+    });
+
+    it("should not be able to create parallel payouts of the same token", async () => {
+      await colony.startNextRewardPayout(otherToken.address);
+
+      await checkErrorRevert(colony.startNextRewardPayout(otherToken.address), "colony-reward-payout-token-active");
+    });
+
+    it("should be able to collect rewards from multiple payouts of different token", async () => {
+      const tokenArgs = getTokenArgs();
+      const newToken = await Token.new(...tokenArgs);
+      await fundColonyWithTokens(colony, newToken, initialFunding.toString());
+
+      const payoutId1 = await colony.startNextRewardPayout.call(newToken.address);
+      await colony.startNextRewardPayout(newToken.address);
```

Comment:
I'd argue to not return anything out of that function and instead get the `payoutId` via emitting an `event` out of the transaction itself. See https://github.com/JoinColony/colonyNetwork/blob/develop/test/colony-network-auction.js#L92